pom.xml

```diff
@@ -205,6 +213,43 @@
 			<artifactId>test-container</artifactId>
 			<version>${project.version}</version>
 		</dependency>
+
+		<!-- Transitive dependency version overrides for Vulnerabilities: -->
+		<!-- overriding version commons-io for vertex 3.9.3 - Vulnerability -->
+		<dependency>
+			<groupId>commons-io</groupId>
+			<artifactId>commons-io</artifactId>
+			<version>2.5</version>
+		</dependency>
+		<!-- overriding version of snakeyaml for prometheus.jmx.collector 0.12.0: Vulnerability: DOS -->
```

Comment:
You changed the JMC Collector version to 0.10 ... so is this still needed?

Reply:
Yes, it is bringing in version 2.4 which has a Remote Code Execution.